prime-router/src/main/kotlin/cli/tests/TestReportStream.kt

```diff
@@ -1984,4 +1985,95 @@ class OtcProctored : CoolTest() {
             return bad("Tests FAILED: " + failures)
         }
     }
+}
+
+class CrudRestApi : CoolTest() {
+    override val name = "CrudRestApi"
+    override val description = "Testing CRUD REST API"
+    override val status = TestStatus.DRAFT
+
+    val dummyOrgObjString = "{" +
```

Comment:
Don't create your own JSON string.  There are libraries for that.  See Json, JsonObject, etc.